src/ck3/data/interactions.rs

```diff
@@ -78,6 +78,14 @@ impl DbKind for CharacterInteraction {
             }
         }
 
+        vd.field_validated_block("localization_values", |block, data| {
+            let mut vd = Validator::new(block, data);
+            vd.unknown_value_fields(|key, value| {
+                let scopes = validate_target(value, data, &mut sc, Scopes::all());
```

Comment:
`this` is possible as the RHS, so use `validate_target_ok_this` instead

Reply:
I think it's a case where `this` makes no sense because in character interactions there's no root

Reply:
Sorry you are correct. I was thinking about the example in health_effects